.changeset/config.json

```diff
@@ -7,13 +7,12 @@
     }
   ],
   "commit": false,
-  "fixed": [["@forgerock/javascript-sdk"]],
+  "fixed": [["@forgerock/javascript-sdk"], ["@forgerock/ping-protect"]],
```

Comment:
This will cause protect to always be versioned along with the js sdk, changing the protect version from 4.6.0 --> 4.8.1 in this release. Is that correct/intended?

Reply:
It won't version it with the SDK, but the SDK would get bumped because it's a dependent.

Reply:
They would have to be grouped in the same sub-array to be fixed together (pretty sure but we can see in the release pr)